multimedia/go2rtc/files/go2rtc.init

```diff
@@ -4,14 +4,54 @@ START=99
 
 USE_PROCD=1
 PROG=/usr/bin/go2rtc
-USER=go2rtc
-GROUP=go2rtc
+CONF=go2rtc
 
 start_service() {
+	local disable_jail
+	local user
+	local group
+	local limit_nofile
+	local allow_config_edit
+	local ycfg=/etc/go2rtc.yaml
+
+	config_load "$CONF"
+	config_get_bool disable_jail daemon disable_jail 0
+	config_get user daemon user go2rtc
+	config_get group daemon group go2rtc
+	config_get limit_nofile daemon limit_nofile ''
+	config_get_bool allow_config_edit daemon allow_config_edit 0
+
+	chown "$user:$group" "$ycfg"
+	if [[ "$allow_config_edit" -ne 0 ]]; then
+		chmod 640 "$ycfg"
+	else
+		chmod 440 "$ycfg"
+	fi
+
 	procd_open_instance
-	procd_set_param command "$PROG" -config /etc/go2rtc.yaml
-	procd_set_param user "$USER"
-	procd_set_param group "$GROUP"
+	procd_set_param command "$PROG" -config "$ycfg"
```

Comment:
I think splitting each argument into a separate line helps with readability, but your call:

```suggestion
	procd_set_param command "$PROG"
	procd_append_param command -config "$ycfg"
```

Reply:
In my opinion appending options is less readable, so i prefer to use it for dynamic params only.